### PR TITLE
Readme updates, vsix updates

### PR DESCRIPTION
**Pull request by swernli**

### CODE REVIEW

**Review by minestarks — APPROVED** [review]

**Review by DmitryVasilevsky — APPROVED** [review]

File: QsCompiler.sln [hunk 1]
```diff
@@ -73,8 +73,6 @@ Project("{6EC3EE1D-3C4E-46DD-8F32-0CC8E7565705}") = "App", "src\QsFmt\App\App.fs
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Telemetry", "src\Telemetry\Library\Telemetry.csproj", "{2A562128-2FD0-47CF-B457-DDF3C19CCDAC}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Tests.LanguageServer", "src\QsCompiler\Tests.LanguageServer\Tests.LanguageServer.csproj", "{A9E3087B-F21A-45D4-A0A8-DF1DB0043233}"
```
Comment (minestarks) [inline QsCompiler.sln:76]: Why did this project get removed?

Comment (swernli) [reply QsCompiler.sln:76]: I'll rever this part.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,14 @@
-# Microsoft Quantum Development Kit: <br>Q# Compiler and Language Server #
+## DEPRECATION NOTICE
 
-Welcome to the Microsoft Quantum Development Kit!
+**This repository is deprecated.** 
+
+For the Modern QDK repository, please visit [Microsoft/qsharp](http://github.com/microsoft/qsharp).
```
Comment (billti) [inline README.md:5]:
```suggestion
For the Modern QDK repository, please visit [Microsoft/qsharp](https://github.com/microsoft/qsharp).

```

File: src/VSCodeExtension/README.md [hunk 3]
```diff
@@ -1,38 +1,11 @@
...
+## DEPRECATION NOTICE
 
-The Quantum Development Kit contains the tools you'll need to build your own quantum computing programs and experiments.
+**This extension is deprecated.** 
 
-Assuming some experience with Visual Studio Code, beginners can write their first quantum program, and experienced researchers can quickly and efficiently develop new quantum algorithms.
+For the latest Q# and Azure Quantum development experience, please use the Azure Quantum Development Kit (Modern QDK) [VS Code extension](https://marketplace.visualstudio.com/items?itemName=quantum.qsharp-lang-vscode).
 
-For version history and related information, please refer to the [Quantum Development Kit Release Notes](https://docs.microsoft.com/en-us/azure/quantum/qdk-relnotes).
+You can also try out the Modern QDK in VS Code for Web at [vscode.dev/quantum](http://vscode.dev/quantum).
```
Comment (billti) [inline src/VSCodeExtension/README.md:9]:
```suggestion
You can also try out the Modern QDK in VS Code for Web at [vscode.dev/quantum](https://vscode.dev/quantum).
```

**Pull request merged**